src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString
+    .split(';')
+    .map((style) => {
+      return style.trim();
+    })
+    .filter((style) => style !== '');
+
+  const styles = styleArray.reduce((prev, style) => {
```

Comment:
Instead of using `reduce` with a spread operator to construct a new object on each iteration, consider using a more efficient approach that modifies the accumulator directly.